[2025-01-27T17:50:51.614-0300] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_data.wait_step_1 manual__2025-01-27T20:50:43+00:00 [queued]>
[2025-01-27T17:50:51.617-0300] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_data.wait_step_1 manual__2025-01-27T20:50:43+00:00 [queued]>
[2025-01-27T17:50:51.617-0300] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-01-27T17:50:51.729-0300] {taskinstance.py:2191} INFO - Executing <Task(ExternalTaskSensor): wait_step_1> on 2025-01-27 20:50:43+00:00
[2025-01-27T17:50:51.734-0300] {standard_task_runner.py:60} INFO - Started process 165482 to run task
[2025-01-27T17:50:51.736-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'load_data', 'wait_step_1', 'manual__2025-01-27T20:50:43+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/step2.py', '--cfg-path', '/tmp/tmp_gbibm7z']
[2025-01-27T17:50:51.737-0300] {standard_task_runner.py:88} INFO - Job 62: Subtask wait_step_1
[2025-01-27T17:50:51.911-0300] {task_command.py:423} INFO - Running <TaskInstance: load_data.wait_step_1 manual__2025-01-27T20:50:43+00:00 [running]> on host victor-MS-7C52
[2025-01-27T17:50:52.100-0300] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='load_data' AIRFLOW_CTX_TASK_ID='wait_step_1' AIRFLOW_CTX_EXECUTION_DATE='2025-01-27T20:50:43+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-27T20:50:43+00:00'
[2025-01-27T17:50:52.100-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:51:22.103-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:51:52.106-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:52:22.108-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:52:52.111-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:53:22.114-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:53:52.117-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:54:22.119-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:54:52.122-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:55:22.202-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:55:52.205-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:56:22.208-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:56:52.210-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:57:22.213-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:57:52.215-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:58:22.218-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:58:52.220-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:59:22.223-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T17:59:52.225-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T18:00:22.228-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T18:00:52.231-0300] {external_task.py:244} INFO - Poking for tasks ['organize_folders'] in dag extract_data on 2025-01-27T20:48:43+00:00 ... 
[2025-01-27T18:00:52.233-0300] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/victor/Desktop/TESTE/code-challenge/.meltano/utilities/airflow/venv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/victor/Desktop/TESTE/code-challenge/.meltano/utilities/airflow/venv/lib/python3.11/site-packages/airflow/sensors/external_task.py", line 334, in execute
    super().execute(context)
  File "/home/victor/Desktop/TESTE/code-challenge/.meltano/utilities/airflow/venv/lib/python3.11/site-packages/airflow/sensors/base.py", line 282, in execute
    raise AirflowSensorTimeout(message)
airflow.exceptions.AirflowSensorTimeout: Sensor has timed out; run duration of 600.132832113999 seconds exceeds the specified timeout of 600.0.
[2025-01-27T18:00:52.236-0300] {taskinstance.py:1138} INFO - Immediate failure requested. Marking task as FAILED. dag_id=load_data, task_id=wait_step_1, execution_date=20250127T205043, start_date=20250127T205051, end_date=20250127T210052
[2025-01-27T18:00:52.331-0300] {standard_task_runner.py:107} ERROR - Failed to execute job 62 for task wait_step_1 (Sensor has timed out; run duration of 600.132832113999 seconds exceeds the specified timeout of 600.0.; 165482)
[2025-01-27T18:00:52.343-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-01-27T18:00:52.356-0300] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
